quill-cassandra-ce/src/main/scala-2.12/io/getquill/CassandraCeContext.scala

```diff
@@ -0,0 +1,101 @@
+package io.getquill
+
+import com.datastax.driver.core._
+import io.getquill.context.cassandra.CqlIdiom
+import io.getquill.util.{ ContextLogger, LoadConfig }
+import cats.effect._
+import cats._
+import io.getquill.util.GuavaCeUtils._
+import cats.implicits._
```

Comment:
```suggestion
import cats.syntax.all._
```

Instances are brought into scope automatically now.